Eastern-Research-Group/csb-rebate-forms-app
# Feature/update epa template

## Related Issues:
* CSBAPP-343

## Main Changes:
* Update the EPA template to the latest version.

## Steps To Test:
1. Navigate to the app.
2. Ensure the EPA footer contains the updated "Accessibility Statement" link
3. Ensure the Twitter logo has been updated to the X logo
